Resources/Prototypes/_LostParadise/Loadouts/Jobs/justice.yml

```
@@ -12,25 +12,36 @@
+#  id: LoadoutLPPWeaponRevolverJudge
+#  category: JobsCommand
+#  cost: 7
+#  exclusive: true
+#  requirements: 
+#    - !type:CharacterJobRequirement
+#      jobs:
+#        - ChiefJustice
+#  items:
+#    - LPPWeaponRevolverJudge
+
+#- type: loadout
```

yamllint

[warning] missing starting space in comment

(comments)